00-Introducing-HOOMD-blue/03-Initializing-the-System-State.ipynb

```diff
@@ -109,7 +109,7 @@
   },
```

Comment:
<p>Is it meant to be *[length]? and *pressure?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>I was experimenting with different formatting. Wolfram Alpha uses the [] for base units in equations. I think using italics in prose and [] in equations makes the definitions clear and readable. Does anyone have another suggestion?</p>